### ISSUE TITLE
Update command console:self-upgrade

### ISSUE THREAD

**Issue by nenes25** [nenes25/prestashop_console#238]
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.6 / 2.0
| Related to command or global ?      | Update console:self-upgrade to manage v1 and v2
| Enhancement description?  | In order to deal with the upgrade of both v1.6.x and v2.x versions this command should be refactored and the whole process of the phar generation too. <br />A good enhancement will be to generate the phar through github actions when making the release.<br />I need to study this point

**Comment by nenes25** [comment]
Not so easy, the current phar upgrader had a github strategy but it doesn't seems to works depsite the fact i have registered the module on packagist.

**Comment by SebSept** [comment]
I've upgraded from 2.0 to ... 1.6.3
Is there a local backup ?

**Comment by SebSept** [comment]
There is one, next to the new  phar : `bin/prestashopConsole-old.phar`

**Comment by nenes25** [comment]
yes the branch 2.0 is waiting for month to be release but i stil have issue with upgrade of both in parallel
Hope to do this soon :smile:
